importer/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views for the importer app.
+"""
+
+from django.shortcuts import render
+from django.contrib.auth.models import User
+
+from learningresources.models import Repository, Course, LearningResource
+from .forms import UploadForm
+
+
+def status(request):
+    """
+    Show what's currently in the system.
+    """
+    counts = {
+        "repos": Repository.objects.count(),
+        "courses": Course.objects.count(),
+        "loxes": LearningResource.objects.count(),
+    }
+    return render(
+        request,
+        "status.html",
+        counts,
+    )
+
+
+def upload(request):
+    """
+    Upload a OLX archive.
+    """
+    form = UploadForm()
+    if request.user.is_anonymous():
+        request.user, _ = User.objects.get_or_create(username="dirty_hack")
```

Comment:
I know this is just getting stuff in here, but as soon as this merges to master, anyone on the Internet will be able to upload large tar balls to our app.  Maybe at least make a note in the doc string that this view should promptly be protected?